kong/core/handler.lua

```diff
@@ -105,11 +109,14 @@ return {
         -- balancer          = nil,            -- the balancer object, in case of a balancer
         -- hostname          = nil,            -- the hostname belonging to the final target IP
       }
+    end,
+    -- Only executed if the `router` module found an API and allows nginx to proxy it.
+    after = function()
+      local ctx = ngx.ctx
+      local var = ngx.var
+      local now = get_now()
```

Comment:
Should we move this call to after the resolution, so we're still factoring in balancer exection time into the below timers?